.github/workflows/OWNERS

```diff
@@ -0,0 +1,10 @@
+options:
+  no_parent_owners: true
+filters:
+  ".*":
+    approvers:
+      - snehakpersistent
+      - mayurwaghmode
+  ".\\e2e-ci-ppc64le.yml":
+      labels:
+        - kind/api-change
```

Comment:
I expected the approvers here. Or maybe I misunderstand the OWNERS file semantics?

Reply:
Compare https://www.kubernetes.dev/docs/guide/owners/